src/Microsoft.AspNetCore.Diagnostics/DeveloperExceptionPage/Views/Obsolete/CompilationErrorModel.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Builder;
+
+namespace Microsoft.AspNetCore.Diagnostics.Views
+{
+    /// <summary>
+    /// Holds data to be displayed on the compilation error page.
+    /// </summary>
+    [Obsolete("message here")]
```

Comment:
(Same in other files)

---
In reply to: [75360275](https://github.com/aspnet/Diagnostics/pull/329#discussion_r75360275) [](ancestors = 75360275,75360256)